Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -2916,21 +2919,24 @@ - (void)yapDatabaseModified:(NSNotification *)notification
                     break;
                 }
                 case YapDatabaseViewChangeMove: {
-                    DDLogVerbose(@"YapDatabaseViewChangeMove: %@, %@, %@",
+                    DDLogVerbose(@"YapDatabaseViewChangeMove: %@, %@, %@, %zd",
                         rowChange.collectionKey,
                         rowChange.indexPath,
-                        rowChange.newIndexPath);
-                    [self.collectionView deleteItemsAtIndexPaths:@[ rowChange.indexPath ]];
-                    [self.collectionView insertItemsAtIndexPaths:@[ rowChange.newIndexPath ]];
```

Comment:
This was the wrong way to do `move` operations; but never we avoid moves anyhow.